core/src/libraries/Positions.sol

```diff
@@ -14,7 +30,7 @@ uint256 constant Q24 = 0x1000000;
  * @param zipped Encoded Uniswap positions. Equivalent to the layout of `int24[6] storage yourPositions`
  * @return positionsOfNonZeroWidth Flattened array of Uniswap positions that may or may not hold liquidity
  */
-function extract(uint144 zipped) pure returns (int24[] memory positionsOfNonZeroWidth) {
+function extract(uint256 zipped) pure returns (int24[] memory positionsOfNonZeroWidth) {
```

Comment:
Yeah; it's because under the hood everything is in 256 bit chunks, and Yul (the language used in in-line assembly blocks) treats everything as uint256.